edm4hep.yaml

```diff
@@ -875,3 +905,4 @@ interfaces:
     Types:
       - edm4hep::TrackerHit3D
       - edm4hep::TrackerHitPlane
+      - edm4hep::SensitiveWireHit
```

Comment:
How confusing will the slightly different meanings of `position` be in this case? Resp. do we need to worry about this?

Reply:
Yes, if used blindly together with the position of the other data types in this interface, things will go wrong (a 3D hit position is ill-defined for this new data type). We went for the less intrusive option w.r.t. existing edm4hep but we can remove the position from the interface and name it differently for the `SensitiveWireHit`, if there is a consensus that this is the way to go. Probably a good thing to discuss tomorrow?